docs/graphql/core/queries/distinct-queries.rst

```diff
@@ -22,6 +22,8 @@ It is typically recommended to use ``order_by`` along with ``distinct_on`` to en
 Note that the ``distinct_on`` column needs to be the first column in the ``order_by`` expression.
 See :ref:`sort queries <sort_query_results>` for more info on using ``order_by``.
 
+Let's say we have the following schema:
```

Comment:
It is correct that we have to introduce the schema here, but what follows below isn't introducing the schema. It's just a reference how to use the `distinct_on` argument. We still have to introduce the schema, such as:

`employee (id integer, name text, department text, salary integer)`

(as an SQL codeblock)